RecoMET/METFilters/python/ecalBadCalibFilter_cfi.py

```diff
@@ -0,0 +1,32 @@
+import FWCore.ParameterSet.Config as cms
+ 
+ecalBadCalibFilter = cms.EDFilter(
+  "EcalBadCalibFilter",
+ 
+  # use this if using AOD:
+  # the Ecal rechit collection found in AOD
+  EcalRecHitSource = cms.InputTag('reducedEcalRecHitsEE'),
+ 
+ 
+  # use this if using MINIAOD:
+  # the Ecal rechit collection found in MINIAOD
+  # EERecHitSource = cms.InputTag('reducedEgamma','reducedEERecHits'),
+ 
+ 
+ 
+  # minimum rechit et to flag as bad: 
+  ecalMinEt        = cms.double(50.),
+  # DetId of bad channel:
+  baddetEcal       = cms.vuint32(),
+
+  from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+  phase2_common.toModify(ecalBadCalibFilter, baddetEcal = [872439604,872422825,872420274,872423218,
```

Comment:
this has to be outside of the scope of the default ecalBadCalibFilter definition
please move this after the last closing brace in this file.
Sorry for being unclear.